### DISCUSSION TITLE
Git plugin: "gl" should be short for "git log", not "git push"

### DISCUSSION THREAD

**Discussion by ivanjonas** [ohmyzsh/ohmyzsh#13528] [Help] [answered]
It seems more intuitive for "gl" to mean "git log", and perhaps that was the intent. In the line below, a grouping of "git log"-related aliases can be seen. "git push" seems out of place.

https://github.com/ohmyzsh/ohmyzsh/blob/51772732f54f01522ad5720e8ed0ba16ca3147ae/plugins/git/git.plugin.zsh#L186-L202

@robbyrussell I tried to find a list of the plugin maintainers but could not find one. Perhaps it could be linked from the contributing guidelines?

**Comment by tryfinally** [comment]
While its seems logical to do the change, it will go against a lot of muscle memory.
Note that `git pull` is used in higher frequency that `git log` and it make sense to give it a shorter alias.
2 chars for the frequently used: `gp` and `gl` (`git push` and `git pull` respectively)
3 or more chars for the less frequently used `git log` variants.
Maybe some one can come with a creative short alias  for the duo `git push` `git pull` in order or enable the transition.

**Comment by KANGOD** [comment]
While for me `git log` is much more frequently used.
Normally using `git fetch` `git rebase` instead of `git pull`.

**Comment by h0adp0re** [comment, marked as answer]
```sh
unalias gl
alias gl="git log"
```

there, everyone can be happy
